src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/help.py

```diff
@@ -87,7 +88,7 @@
                     az lab vm create --lab-name MyLab -g MyRG --name MyVM --image "Ubuntu Server 16.04 LTS" --image-type gallery --size Standard_DS1_v2 --ip-configuration public
                 - name: Create a Virtual Machine from a formula.
                   text: >
-                    az lab vm create --lab-name MyLab -g MyRG --name MyVM --formula MyFormula
+                    az lab vm create --lab-name MyLab -g MyRG --name MyVM --formula MyFormula --artifacts @/artifacts.json
```

Comment:
Does this PR fix the fact that if you omit --artifacts it should use the formula's defaults?